pkg/state/state.go

```diff
@@ -1629,6 +1629,14 @@ func (s *stateManager) AddrByAlias(alias proto.Alias) (proto.WavesAddress, error
 	return *addr, nil
 }
 
+func (s *stateManager) AliasesByAddr(addr *proto.WavesAddress) ([]string, error) {
```

Comment:
Use just `proto.WavesAddress`, not `*proto.WavesAddress`. Pointer can be `nil`, struct can't.